src/Elastic.Markdown/HtmlWriter.cs

```diff
@@ -59,18 +60,23 @@ private async Task<RenderResult> RenderLayout(MarkdownFile markdown, MarkdownDoc
 	{
 		var html = MarkdownFile.CreateHtml(document);
 		await DocumentationSet.ResolveDirectoryTree(ctx);
-		var navigationItem = DocumentationSet.FindNavigationByMarkdown(markdown);
+		var navigationItem = NavigationTraversable.GetNavigationItem(markdown);
```

Comment:
Q: Is this already the "current" navigation item?

What's the difference between `NavigationTraversable.GetNavigationItem` and `NavigationTraversable.GetCurrent`